src/transformers/models/idefics/clip.py

```diff
@@ -318,16 +318,16 @@ def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
         return hidden_states
 
 
-# copied from transformers.models.clip.modeling_clip.CLIPEncoderLayer
 class CLIPEncoderLayer(nn.Module):
-    def __init__(self, config: CLIPConfig):
+    def __init__(self, config: CLIPVisionConfig):
```

Comment:
same here - I thought we are replacing CLIPVisionConfig with IdeficsVisionConfig, no?